packages/notifi-dapp-example/src/assets/Icon.tsx

```diff
@@ -16,7 +16,8 @@ export type SpriteIconId =
   | 'email-icon'
   | 'slack-icon'
   | 'telegram-icon'
-  | 'discord-icon';
+  | 'discord-icon'
```

Comment:
I still can see the following files in `public/logo/`, shall we remove those?
- discord-icon.svg
- email-icon.svg
- telegram-icon.svg
- slack-icon.svg